github/ipfs.yml

```diff
@@ -764,8 +764,10 @@ repositories:
           *       @ipfs/ipdx
     teams:
       maintain:
-        - w3dt-stewards
         - ipdx
+      push:
+        - org-admins
```

Comment:
Or maybe instead we should only ever add collaborators directly to this repository and disallow teams altogether?

Reply:
I thought about it more. With a team we can use `CODEOWNERS` with auto assigned reviewers feature more effectively. In my opinion, that's good enough reason to stick with a team setup. Let me just add some more comments and I'll make this one ready for review.